provider/lxd/environ_raw.go

```diff
@@ -135,7 +130,7 @@ func getRemoteConfig(spec environs.CloudSpec) (*jujulxdclient.Config, error) {
 	}, nil
 }
 
-func getCerts(spec environs.CloudSpec) (client *jujulxdclient.Cert, server string, ok bool) {
+func getCerts(spec environs.CloudSpec) (client *lxd.Certificate, server string, ok bool) {
```

Comment:
nit: if we're renaming cert to certificate elsewhere, perhaps this could get getCertificates

Reply:
I'll make this change.